app/models/api/fixed_region.rb

```diff
@@ -0,0 +1,18 @@
+require 'dry-validation'
```

Comment:
The idea is that for each "resource" we are expecting to get via http we just create a new file like this with
- validation / schema
- coercion / struct

The rest of the PR is reusable infrastructure